pod/perldiag.pod

```diff
@@ -2453,6 +2453,12 @@ some time before now.  Check your control flow.  flock() operates on
 filehandles.  Are you attempting to call flock() on a dirhandle by the
 same name?
 
+=item for my (...) is experimental
+
+(S experimental::for_list) This warning is emitted if you use C<for> to
+iterate over a list of lexicals. This syntax is currently experimental and
```

Comment:
`to iterate over a list of lexicals` is confusing to me and I always think you mean:
`for my $v ($lexical1, $lexical2, $lexical3) {.. }` but maybe that's just me

Reply:
Good catch. I've changed it to this:

```
(S experimental::for_list) This warning is emitted if you use C<for> to
iterate multiple values n-at-a-time. This syntax is currently experimental
and its behaviour may change in future releases of Perl.
```

because *n-at-a-time* is how it's described in `perlsyn.pod`.